core/src/cats/make_catalog_backup.pl.in

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env perl
+#!/usr/bin/perl
```

Comment:
you're not wrong, but we will have to support systems that have perl in unusual places (i.e. FreeBSD has it in /usr/local/bin/perl).
So far /usr/bin/env has worked well. We could change this, but we would have to configure the path system specific and detect the path to perl in CMake and set it using a variable.
If you propose a patch, I'll test and implement it.

Having said that, we're trying to remove/replace all perl scripts anyways, so I'm not really sure it is worth the effort anymore.